pkg/executor/build.go

```diff
@@ -701,11 +701,16 @@ func fetchExtraStages(stages []config.KanikoStage, opts *config.KanikoOptions) e
 				continue
 			}
 			// Check if the name is the alias of a previous stage
+			previousName := false
 			for _, name := range names {
 				if name == c.From {
-					continue
```

Comment:
Definitely not a golang expert here, but pretty sure you could use a [`continue` statement with label](https://golang.org/ref/spec#Continue_statements) instead of a boolean flag.

Reply:
@gilbsgilbs I am not a golang expert either, however I've been instructed not to ever use a label .. Out of machine language, I guess. A better solution would be to isolate this pied of code in a dedicated function I guess

Reply:
I isolated this code in a dedicated function. Great profile picture by the way :smile: